src/ui/RecipeTitleCard.tsx

```diff
@@ -0,0 +1,103 @@
+import styled from '@emotion/styled';
+import Flex from './Flex';
+
+const StyledFooter = styled.footer`
+  display: flex;
+  gap: 8px;
+`;
+
+const BydelName = styled.footer`
+  font-size: 17px;
+  line-height: 20.74px;
+  font-weight: 600;
+  // font-family-sneak;
+`;
+
+const RecipeName = styled.label`
+  font-weight: 600;
+  text-transform: uppercase;
+  font-size: 30px;
+  line-height: 34.2px;
+`;
+const AuthorName = styled.label`
+  width: 277px;
+  font-weight: 400;
+  text-transform: uppercase;
+  font-size: 30px;
+  line-height: 34.2px;
+  vertical-align: center;
+`;
+
+const StyledFlex = styled(Flex)`
+  width: 365px;
+  height: 189px;
+  background: var(--pink);
+  color: #14000f;
+  align-content: center;
+  padding: 30px;
+  gap: 10px;
+  margin-bottom: 10px;
+  letter-spacing: 0.01em;
+  justify-content: space-around;
```

Comment:
if it's a prop on the Flex component, you should put it in `<StyledFlex justifyContent="space-around" />` instead of here